pipeline/templates/unsubscribe/unsubscribe_website.html

```diff
@@ -57,12 +57,39 @@
             gap: 40px;
         }
     </style>
+     <script>
+        document.addEventListener('DOMContentLoaded', function() {
+            var form = document.querySelector('form');
+            var submitButton = form.querySelector('button[type="submit"]');
+
+            submitButton.disabled = true; 
+
+            function checkFormValidity() {
+                var isValid = form.checkValidity();
+                submitButton.disabled = !isValid;
+            }
+
+            Array.from(form.elements).forEach(function(element) {
+                element.addEventListener('input', checkFormValidity);
+            });
+        });
+    </script>
 </meta>
 </meta>
 <body>
-    <h1>Unsubscribe</h1>
+    <h1>Unsubscribe Homepage</h1>
     <p>Please enter your email to find your subscribed products.</p>
 
```

Comment:
The use of different pages is amazing!

Looking forward to the checkboxes for unsubscribing.